pkgs/misc/emulators/commander-x16/emulator.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, lib
+, fetchFromGitHub
+, SDL2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "x16-emulator";
+  version = "38";
+
+  src = fetchFromGitHub {
+    owner = "commanderx16";
+    repo = pname;
+    rev = "r${version}";
+    sha256 = "WNRq/m97NpOBWIk6mtxBAKmkxCGWacWjXeOvIhBrkYE=";
+  };
+
+  dontConfigure = true;
+
+  buildInputs = [ SDL2 ];
+
+  installPhase = ''
+    runHook preInstall
+    install -D --mode 755 --target-directory $out/bin/ x16emu
+    install -D --mode 444 --target-directory $out/share/doc/${pname} README.md
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
meta should be last.

Reply:
Usually, meta is the last attribute.
However, in the rare cases when `passthru` is used, it comes last.

(The idea of passthru here is in the case someone creates a wrapping script encapsulating the emulator and the rom; according to the upstream site, they are updated at the same time, and older ROMs may not work with newer emulators.)